Demo/Components/VisibilityDrivenTitleView/VisibilityDrivenTitleDemoView.swift

```diff
@@ -4,25 +4,23 @@
 
 import FinniversKit
 
-class PercentageDrivenTitleDemoView: UIView {
+class VisibilityDrivenTitleDemoView: UIView {
 
     // MARK: - Private properties
 
-    private lazy var titleView: PercentageDrivenTitleView = {
-        let view = PercentageDrivenTitleView(withAutoLayout: true)
-        view.title = "Look ma I can hide"
+    private lazy var titleView: VisibilityDrivenTitleView = {
+        let view = VisibilityDrivenTitleView(withAutoLayout: true)
+        view.title = "Now you see me"
         view.font = UIFont.title3Strong
-        view.setPercentageVisible(1)
+        view.setIsVisible(true)
         return view
     }()
 
-    private lazy var slider: UISlider = {
-        let slider = UISlider(withAutoLayout: true)
-        slider.minimumValue = 0
-        slider.maximumValue = 1
-        slider.setValue(1, animated: false)
-        slider.addTarget(self, action: #selector(sliderDidChange), for: .valueChanged)
-        return slider
+    private lazy var visibilitySwitch: UISwitch = {
```

Comment:
Why do we need this switch here?

Reply:
Just to have some form of interaction with the view